adapters/repos/db/index.go

```diff
@@ -2885,3 +2885,25 @@ func (i *Index) DebugRepairIndex(ctx context.Context, shardName, targetVector st
 
 	return nil
 }
+
+func (i *Index) DebugRequantizeIndex(ctx context.Context, shardName, targetVector string) error {
+	shard, release, err := i.GetShard(ctx, shardName)
+	if err != nil {
+		return err
+	}
+	if shard == nil {
+		return errors.New("shard not found")
+	}
+	defer release()
```

Comment:
`release()` will be called before the end of the `RequantizeIndex`, we should call it in the goroutine instead

Reply:
I'll add cancelation for this endpoint and RepairIndex in another PR